FSJ_django20_project/FSJ/views.py

```diff
@@ -245,4 +245,84 @@ def view_application(request):
     context["archived"] = False
     template = loader.get_template("FSJ/view_application.html")
     return HttpResponse(template.render(context, request))
-    
\ No newline at end of file
+
```

Comment:
Everything in this file would normally belong in views_coordinator.py given that the functionality is exclusive to the coordinator. However, I don't understand why it exists at all since it's just a copy of the existing edit methods with a new name and the method signature is missing the argument that is instead being accessed as a parameter from the GET. Why not just modify the existing methods to remove the extra argument, change the method name, and add the line getting the parameter?